Extensions/one_click_postage.js

```diff
@@ -696,18 +696,17 @@ XKit.extensions.one_click_postage = new Object({
 		};
 
 		var menu_close = function() {
-			// Only close the menu if it doesn't have keyboard or mouse focus
-			if ($("#x1cpostage_box").find('input:focus, textarea:focus').length === 0 &&
-				$('#x1cpostage_box:hover').length === 0) {
```

Comment:
why remove the hover selector? This seems to say that as soon as you deselect one text area, the menu will close.

Reply:
In practice, at least with Firefox, `#x1cpostage_box:hover` will match the popup whilst we're handling `mouseleave`. Your options for defocussing any of the inputs are: (i) focus on another input; (ii) click on the fake radio control; or (iii) click on one of the trigger buttons. Handlers exist to cancel the timeout for the first two cases, and the third case is a non-issue.

Reply:
debounce?

Reply:
That's what the original timer mechanism is for.